app/src/main/res/values/strings.xml

```diff
@@ -432,7 +432,10 @@
     <string name="profile_settings">Profile Settings</string>
     <string name="loading_tickets">Loading tickets...</string>
     <string name="download_invoice">Download Invoice</string>
-    <string name="invalid_url_message">This is not a valid URL</string>
+    <string name="invalid_url_message">Please enter the correct username</string>
+    <string name="facebookHeadUrl">https://www.facebook.com/</string>
+    <string name="twitterHeadUrl">https://www.twitter.com/</string>
+    <string name="instagramHeadUrl">https://www.instagram.com/</string>
```

Comment:
These are untranslatable strings and shouldn't be present here

Reply:
where should I keep them then?

Reply:
I am adding `translatable="false"` tag to them and moving them at the bottom